lib/engines/bh.js

```diff
@@ -7,37 +7,49 @@ module.exports = function (opts) {
 
     /**
      * bh.js express view engine adapter
-     * @api
-     * @returns {function(string, Object, Function)}
+     * Produces html from bh-templates and data and invokes passed callback with result
+     *
+     * @param   {String}     name       bundle name
+     * @param   {Object}     [options]  data to use in templates
+     * @param   {Function}   cb         callback to invoke with result or error
+     * @returns {Undefined}             undefined value
      */
     function bh(name, options, cb) {
+        var filePath;
+        var bhModule;
+        var html;
+
         // reject rendering for empty options.bemjson
         if (!options.bemjson) {
-            cb(Error('Empty request'));
+            cb(Error('No bemjson was passed'));
             return;
         }
 
-        name = U.fulfillName(name, opts.ext || this.ext, opts.source);
+        filePath = U.fulfillName(name, opts.ext || this.ext, opts.source);
 
-        // pass to render
+        // check cache options
         if (opts.force || options.forceExec || options.forceLoad) {
             dropRequireCache(require, name);
         }
 
-        var bhModule;
         try {
-            bhModule = require(name);
+            bhModule = require(filePath);
         } catch (e) {
             cb(e);
             return;
         }
 
-        if (!bhModule || !bhModule.apply) {
-            cb(new Error('Unknown file format'));
+        // add data to use in templates
+        options.bemjson.data = bhModule.utils.extend({}, options);
```

Comment:
It's a real hardcode ;-) I'm sure in it. Can we make an option here to pass `options` as some variable into `bemjson`?

It will looks like:
```js
  bem.usePlugin(EXPRESSBH, {dataKey: 'data'});
```
and this block of code will be like:
```js
  if (opts.dataKey) {
      options.bemjson[opts.dataKey] = bhModule.utils.extend({}, options);
      delete options.bemjson[dataKey].bemjson;
  }
```